api-src/org/labkey/api/targetedms/TargetedMSFolderTypeListener.java

```diff
@@ -0,0 +1,10 @@
+package org.labkey.api.targetedms;
+
+import org.labkey.api.data.Container;
+import org.labkey.api.security.User;
+
+public interface TargetedMSFolderTypeListener
```

Comment:
Added so that the PanoramaPublic module can add itself to new TargetedMS folders.